src/server/cluster_config_test.cc

```diff
@@ -27,4 +41,79 @@ TEST_F(ClusterConfigTest, KeyTagTest) {
   ASSERT_EQ(key, ClusterConfig::KeyTag(key));
 }
 
+TEST_F(ClusterConfigTest, ConfigEmpty) {
+  // Test that empty-initialization causes none of the slots to be owned locally.
+  for (SlotId i : {0, 1, 10, 100, 1'000, 10'000, 16'000, 0x3FFF}) {
+    EXPECT_FALSE(config_.IsMySlot(i));
```

Comment:
please add a test where you check that IsMySlot is true and for some false

Reply:
That's a good idea!
I changed `ClusterConfigTest.ConfigSetMultipleInstances` to have that.